src/main.c

```diff
@@ -2832,7 +2836,9 @@ int main(int argc, char **argv) {
             // RENDER 3-D SCENE //
             glClear(GL_COLOR_BUFFER_BIT);
             glClear(GL_DEPTH_BUFFER_BIT);
-            render_sky(&sky_attrib, player, sky_buffer);
+                //
+                render_sky(&sky_attrib, player, sky_buffer);
```

Comment:
What's going on here?

Reply:
Oops. I was using the indentation as a note to myself that I forgot about.